server/database/models/UserLab.js

```diff
@@ -0,0 +1,20 @@
+module.exports = (sequelize, DataTypes) => {
+    const UserLab = sequelize.define(
+        'userlab',
+        {
+            labid: {
+                type: DataTypes.INTEGER,
+                primaryKey: true
+            },
+            userid: {
+                type: DataTypes.INTEGER,
+                primaryKey: true
+            },
```

Comment:
Is this table created yet? I wonder if declaring two primary keys in sequelize would result in an error.

Reply:
Yes you can put two primary keys. Sequelize will create a composite key